src/Umbraco.Core/Persistence/Repositories/ContentTypeBaseRepository.cs

```diff
@@ -101,7 +101,7 @@ protected IEnumerable<int> PerformGetByQuery(IQuery<PropertyType> query)
 
             var translator = new SqlTranslator<PropertyType>(sqlClause, query);
             var sql = translator.Translate()
-                                .OrderBy<PropertyTypeDto>(x => x.PropertyTypeGroupId, SqlSyntax);

```

Comment:
Well as I pointed out in the issue - I had no idea what this task is about or why it was needed as the issue didn't have any context information at all .. it makes a bit more sense now what actually needs to be done :)